torchx/runner/api.py

```diff
@@ -554,7 +554,12 @@ def list(
         """
         with log_event("list", scheduler):
             sched = self._scheduler(scheduler)
-            return sched.list()
+            app_handles = []
+            app_ids = sched.list()
+            for app_id in app_ids:
```

Comment:
We could use list comprehension to build `app_ids` for more idiomatic code:
`
app_handles = [make_app_handle(scheduler, self._name, app_id) for app_id in sched.list()]
`

Reply:
Thanks for the suggestion. Updated!